uber/models/__init__.py

```diff
@@ -1514,6 +1514,9 @@ def assign(self, attendee_id, job_id):
             if not job.no_overlap(attendee):
                 return 'This volunteer is already signed up for a shift during that time'
 
+            if not job.working_limit_ok(attendee):
```

Comment:
I can't find the issue in JIRA (as usual), but this fixes a bug where the assign page would say someone is already signed up for a shift when what it meant is that the max consecutive hours would be surpassed.